OpenSim/Common/DataTable.h

```diff
@@ -843,8 +843,9 @@ class DataTable_ : public AbstractDataTable {
         const size_t totalWidth{std::accumulate(columnWidths.cbegin(),
                                                 columnWidths.cend(),
                                                 static_cast<size_t>(0))};
-        result.reserve((totalWidth * table.size() * excessAllocation) +
-                       result.capacity());
+        result.reserve(result.capacity() +
+            static_cast<unsigned>(std::roundf(
+                totalWidth * table.size() * excessAllocation)));
```

Comment:
> Feel free to add any documentation you think is helpful/necessary.

I have been unable to find a reference suggesting 1.25. The only other time I have looked into memory allocation is for Maple (not a "real" programming language, I admit), which allocates using a factor of 2. Can you point me to the reference you are using?

> My understanding is this is certainly not a bug.

Ok, I have removed `roundf()`.